### PR TITLE
Adding support for insert-host-certificates

### PR DESCRIPTION
**Pull request by stevenbrockman**
This code looks for PEM files under:
> ml-config/security/certificate-templates/hostcertificates/TEMPLATE-NAME

These files MUST be in PEM format, and be named with "crt" and "key" extensions.  A matching pair must exist.   Ex:

> host1.marklogic.com.pem
> host1.marklogic.com.key

Command can be invoked individually on just one template by constructing the Command with the templateName.

### CODE REVIEW

File: src/main/java/com/marklogic/appdeployer/command/databases/DeployDatabaseCommand.java [hunk 1]
```diff
@@ -141,7 +141,7 @@ public void undo(CommandContext context) {
 
     /**
      * Creates and attaches sub-databases to a the specified database, making it a super-database.
-     * Note: Sub-databases are expected to have a configuration files in databases/subdatabases/<super-database-name>
```
Comment (stevenbrockman) [inline src/main/java/com/marklogic/appdeployer/command/databases/DeployDatabaseCommand.java]: compiling on my mac, gave errors with these "<" characters

**Comment by rjrudin** [comment]
@stevenbrockman This looks good - do you have a sample certificate that I can use for testing?

**Comment by stevenbrockman** [comment]
Test Certificates attached (along with bogus CA)

**Comment by rjrudin** [comment]
Thanks @stevenbrockman for the files, I'm trying this out locally now.

**Comment by rjrudin** [comment]
I just tested it out locally, works great. Merging this into a branch, and I'll add a JUnit test for it.

**Pull request merged**